erpnext/manufacturing/doctype/bom/bom.py

```diff
@@ -475,6 +478,7 @@ def get_bom_items_as_dict(bom, company, qty=1, fetch_exploded=1, fetch_scrap_ite
 				item.default_warehouse,
 				item.expense_account as expense_account,
 				item.buying_cost_center as cost_center
+				{select_conditions}
```

Comment:
these are not conditions, these are columns, better name would be `select_columns`